clean_data.py

```diff
@@ -43,30 +43,40 @@ def replace_mrn(df, key: str = 'ReplaceMe') -> pd.DataFrame:
 
 def classify_hearing_loss(df: pd.DataFrame):
   # Assign result to a temporary dataframe, since classifier creates lots of 
-  # temporary column names.
+  # temporary column names.  
 
   orig_df = df.copy()
   new_df = clusters.HLossClassifier(df)
   df = orig_df
-  df['R_Type_HL_Mod'] = new_df['R_Type_HL_Mod']
-  df['R_Type_HL_HF'] = new_df['R_Type_HL_HF']
-  df['R_Type_HL_All'] = new_df['R_Type_HL_All']
+  
+  # Specify columns related to bone conduction, hearing loss types, and PTA to be transferred
+  bc_columns = ['RBone250', 'RBone500', 'RBone1000','RBone2000', 'RBone4000', 
+              'LBone250', 'LBone500', 'LBone1000', 'LBone2000', 'LBone4000']
```

Comment:
Looks like this line isn't properly indented.  Add two spaces?  Similarly below for hl_type_columns.